.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 15
+      matrix:
+        php-versions:  ['7.4', '8.0']
+    name: tests-${{ matrix.php-versions }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@master
+      - name: cache-composer
+        uses: actions/cache@v2
+        with:
+          path: ~/.composer/cache/
+          key: composer-${{ matrix.php-versions }}-${{ github.sha }}
+      - name: setup
+        uses: shivammathur/setup-php@2.9.0
+        with:
+          php-version: ${{ matrix.php-versions }}
+      - name: composer install
+        run: composer update --prefer-stable
+      - name: Build docker
+        run: docker build -t httpbin .
+      - name: Run docker
+        run: docker run -d -p 8081:80 httpbin
+      - name: tests
+        run: composer test
+  tests-lowest:
```

Comment:
Can it be "merged" with the main job?
see https://github.com/jolicode/JoliNotif/blob/main/.github/workflows/ci.yml#L23-L35 for an example